about.html

```diff
@@ -34,114 +34,25 @@ <h2>Project Leads</h2>
-<h5>Aspiring Parkourist<br>Sand Cat Impersonator</h5>
-  </ul>
-</section>
-
-<section id="advisors">
-<h2>Advisors</h2>
-  <ul class="advisors">
-    <li class="swetland">
-      <h3>Brian Swetland</h3>
-      <h4>Operating Systems Expert</h4>
-      <p>As Android's Systems / Kernel Lead from 2004 to 2012, Brian built the lower levels of what is now the world's most-popular operating system. Prior to that, Brian worked as a kernel engineer for BeOS and Danger, Inc.
-    <li class="lutomirski">
-      <h3>Andrew Lutomirski</h3>
-      <h4>Kernel Security Expert</h4>
-      <p>Andy is a Linux kernel developer focused on security. Every month or two, he sends us a message: "I found another Linux kernel vulnerability, but Sandstorm's seccomp filter blocks it." That seccomp filter? Andy wrote it.
-    <li class="nagra">
-      <h3>Jasvir Nagra</h3>
-      <h4>Web Security Expert</h4>
-      <p>Jas is a web security expert, formerly of the Google security team, where among other things he led the <a href="https://developers.google.com/caja/">Caja</a> project to create a variant of Javascript that enforces capability-based security.
-    <li class="seaborn">
-      <h3>Mark Seaborn</h3>
-      <h4>Linux Sandboxing Expert</h4>
-      <p>Mark is a Linux kernel security and sandboxing expert on the Google Chrome team, where among other things he helped build the NaCl sandbox and <a href="http://googleprojectzero.blogspot.com/2015/03/exploiting-dram-rowhammer-bug-to-gain.html">developed the first working exploit of the Rowhammer hardware security flaw</a>. Previous to Google, Mark developed <a href="http://plash.beasts.org/contents.html">Plash</a>, a capability-based sandbox for Unix shell commands.
-    <li class="miller">
-      <h3>Mark S. Miller</h3>
-      <h4>Distributed Security Expert</h4>
-      <p>Mark is one of the leading experts -- arguably <i>the</i> leading expert -- in the field of capability-based security. He is the designer of the <a href="http://erights.org/">E Language</a> and the <a href="http://erights.org/elib/distrib/captp/index.html">CapTP protocol</a> on which <a href="https://capnproto.org">Cap'n Proto</a> (Sandstorm's underlying communications layer) is based.
-  </ul>
+<p>In 2020, a group of Sandstorm enthusiasts began a community effort to <a href="https://sandstorm.io/news/2020-02-03-reviving-sandstorm">revive development</a> of Sandstorm. Ian Denhardt did a significant amount of work improving the experience of authorizing network requests from Sandstorm apps, refactoring older code, and making the codebase more maintainable. In 2021, this community group <a href="https://sandstorm.io/news/2021-06-17-sandstorm-community-fundraising">participated in FundOSS</a> and raised $5,319 from 51 backers to fund additional development in Sandstorm. As of 2022, Sandstorm Development Group has been completely dissolved, and development of the Sandstorm project is transitioning fully to a community model.</p>

```

Comment:
At this point is it fully transitioned? Or is it still transitioning?

Reply:
Technically the latter, but I think we should write it like the former. Because the nuance of the difference is small.